tools/lint/lint.py

```diff
@@ -57,13 +60,19 @@ def setup_logging(prefix=False):
     global logger
     if logger is None:
         logger = logging.getLogger(os.path.basename(os.path.splitext(__file__)[0]))
-        handler = logging.StreamHandler(sys.stdout)  # type: logging.Handler
-        # Only add a handler if the parent logger is missing a handler
+        # Only add a stdout handler if the parent logger is missing a handler
         parent = logger.parent
         assert isinstance(parent, logging.Logger)
         if parent and len(parent.handlers) == 0:
-            handler = logging.StreamHandler(sys.stdout)
+            handler = logging.StreamHandler(sys.stdout)  # type: logging.Handler
             logger.addHandler(handler)
+
+        # When running on Taskcluster, we want to dump to the GitHub Checks
+        # output file too.
+        if os.getenv("TASKCLUSTER_ROOT_URL"):
```

Comment:
I'd rather this was part of a command line option or something.

Reply:
Again, agreed; just playing around :)